pkg/landscaper/installations/imports/constructor.go

```diff
@@ -64,7 +68,7 @@ func (c *Constructor) Construct(ctx context.Context, inst *installations.Install
 }
 
 // constructImports is an auxiliary function that can be called in a recursive manner to traverse the tree of conditional imports
-func (c *Constructor) constructImports(importList lsv1alpha1.ImportDefinitionList, importedDataObjects map[string]*dataobjects.DataObject, importedTargets map[string]*dataobjects.Target, templatedDataMappings map[string]interface{}, fldPath *field.Path) (map[string]interface{}, error) {
+func (c *Constructor) constructImports(importList lsv1alpha1.ImportDefinitionList, importedDataObjects map[string]*dataobjects.DataObject, importedTargets map[string]*dataobjects.Target, importedTargetLists map[string]*dataobjects.TargetList, templatedDataMappings map[string]interface{}, fldPath *field.Path) (map[string]interface{}, error) {
```

Comment:
can you restructure the imports list to be more readable on smaller screens:)
```suggestion
func (c *Constructor) constructImports(
  importList lsv1alpha1.ImportDefinitionList, 
  importedDataObjects map[string]*dataobjects.DataObject, 
  importedTargets map[string]*dataobjects.Target, 
  importedTargetLists map[string]*dataobjects.TargetList, 
  templatedDataMappings map[string]interface{}, 
  fldPath *field.Path) (map[string]interface{}, error) {
```

Reply:
ah, always those people who don't use ultrawide monitors ;-)